src/components/MetadataSources/SourceManagement/Contact/ContactField.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get, isEmpty } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+
+import {
+  Col,
+  Row,
+  Select,
+  TextField,
+} from '@folio/stripes/components';
+
+import FindContact from '../FindContact/FindContact';
+import FindUser from '../FindUser/FindUser';
+import { Required } from '../../../DisplayUtils/Validate';
+
+export default class ContactField extends React.Component {
+  static propTypes = {
+    index: PropTypes.number.isRequired,
+    input: PropTypes.shape({
+      name: PropTypes.string.isRequired,
+    }).isRequired,
+    intialContact: PropTypes.object,
+    selectContact: PropTypes.func.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    this.inputRef = React.createRef();
+
+    const intialContact = props.intialContact || {};
+
+    this.state = {
+      contact: intialContact,
```

Comment:
Is the _contact_ state used somewhere? You pass it through to _FindUser_ and _FindContact_. But as I can see, the state does not change in this component, and the passed through property is not used. If this is true, you can delete the state and the _initialContact_ property.

Reply:
initialContact has been removed